src/imitation/algorithms/adversarial/common.py

```diff
@@ -240,6 +256,7 @@ def __init__(
 
     @property
     def policy(self) -> policies.BasePolicy:
+        # TODO(juan) either change the return type to optional or add an assertion.
```

Comment:
probably an assertion? policy should always be non-None unless the algorithm is not fully initialized